internal/transform/effects.go

```diff
@@ -1278,3 +1295,212 @@ func (e *effectsWrapper) addLiquidityPoolWithdrawEffect() error {
 	e.addMuxed(e.operation.SourceAccount(), EffectLiquidityPoolWithdrew, details)
 	return nil
 }
+
+// addInvokeHostFunctionEffects iterates through the events and generates
+// account_credited and account_debited effects when it sees events related to
+// the Stellar Asset Contract corresponding to those effects.
+func (e *effectsWrapper) addInvokeHostFunctionEffects(events []contractevents.Event) error {
+	if e.operation.network == "" {
+		return errors.New("invokeHostFunction effects cannot be determined unless network passphrase is set")
```

Comment:
FYI the network passphrase needs to be passed to the transformEffects function. I don't think it's passed rn so this will always error